applications/create-substrate-app.md

```diff
@@ -0,0 +1,117 @@
+  - Integration: Works in tandem with the core Substrate framework and the Polkadot/Kusama ecosystem facilitating easier onboarding of Web2 developers. 
+- Who is your target audience (parachain/dapp/wallet/UI developers, designers, your own user base, some dapp's userbase, yourself)?
+  - Primary: Dapp developers new to Substrate, or experienced developers looking to rapidly prototype Substrate projects.
+  - Secondary: Frontend developers within existing Substrate teams wanting standardized project structures.
+- What need(s) does your project meet?
+  - Lowers Barrier to Entry: Reduces the initial learning curve and complexity associated with setting up a Substrate project, especially for those unfamiliar with the framework's specifics.
+  - Standardization: Promotes consistent project structures and best practices within the Substrate development community.
+  - Efficient Development: Saves time by automating project scaffolding and integration of frontend frameworks with chain selection
+- How did you identify these needs? Please provide evidence in the form of (scientific) articles, forum discussions, case studies, or raw data.
+  - Questions on how to get started with building a Substrate dApp are frequently asked in the Substrate Technical chat and on Stack Overflow. There is a clear desire and need for a steamlined tooling, inspired by the popular `create-react-app`. 
+- Are there any other projects similar to yours in the Substrate / Polkadot / Kusama ecosystem?
+  - There are fragmented repos that can be found on Github, but they're mostly on React and need references to a private repo. By adding this global CLI command, we can easily share project scaffolding on tutorials and documentation.  
+
+## Team
+### Team members
+
+- Name of team leader: Tuguldur Baigalmaa
+- Names of team members: Tuguldur Baigalmaa
+
+### Contact
+
+- **Contact Name:** Tuguldur Baigalmaa
+- **Contact Email:** tgldr0511@gmail.com
+- **Website:** https://philoniare.com/about
+
+### Legal Structure
+
+- **Registered Address:** None
+- **Registered Legal Entity:** None
+
+### Team's experience
+
+I have been a web2 developer for 7 years and have maintained web3 js libraries like [terra.js](https://terra-money.github.io/terra.js/)
+
+### Team Code Repos
+
+https://github.com/philoniare/pba-final-project
+
+https://github.com/philoniare/
+
+### Team LinkedIn Profiles (if available)
+
+- https://linkedin.com/in/tuguldurb
+
+
+## Development Status
+
+There's already an [existing template](https://github.com/substrate-developer-hub/substrate-front-end-template). We will be building on top of this template and adding more features to it.
+
+## Development Roadmap :nut_and_bolt:
+
+### Overview
+
+- **Total Estimated Duration:** 1 month
+- **Full-Time Equivalent (FTE):**  1 FTE
+- **Total Costs:** 5,000 USD
+
+### Milestone 1 - Full CLI functionality
+
+- **Estimated duration:** 1 month
+- **FTE:**  1
+- **Costs:** 5,000 USD
```

Comment:
Since JS is no longer supported, could you update this section as well?

Reply:
yep updated, thanks